src/datatypes/src/types/time_type.rs

```diff
@@ -217,4 +225,60 @@ mod tests {
             TimeNanosecondType.as_arrow_type()
         );
     }
+
+    #[test]
+    fn test_time_cast() {
```

Comment:
Question: the time type can be cast from string values?

Reply:
no, it can't.